Tasks-Maps/src/Task_1/AmountOfKeys.java

```diff
@@ -0,0 +1,24 @@
+package Task_1;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AmountOfKeys {
+    public static void main(String[] args) {
+
+    }
+/* 1. Дана Map<String, String> map, написать функцию, которая вернет Мап,
+      такую, если в исходной map есть ключ ‘a’ и ‘b’, то нужно создать новый ключ ‘ab’ с суммой значений от ключей a и b.
+      Примеры:
+      mapAB({"a": "Hi", "b": "There"}) → {"a": "Hi", "ab": "HiThere", "b": "There"}
+      mapAB({"a": "Hi"}) → {"a": "Hi"}
+      mapAB({"b": "There"}) → {"b": "There"}
+*/
+
+    public static Map<String, String> newMap(Map<String, String> hashMap) {
+        if (hashMap.containsKey("a") && hashMap.containsKey("b")) {
+            hashMap.put("ab", "HiThere");
```

Comment:
This is wrong. The condition says that the value for the "ab" key should be a concat of hashMap.get("a")+hashMap.get("b"), not a "HiThere" (it was just an example).